docs/sources/setup-grafana/configure-security/secret-scan.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Detect and revoke leaked Grafana service account tokens
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Configure Grafana secret scanning and notifications
+menuTitle: Configure secret scanning
+weight: 1000
+---
+
+# Configure Grafana secret scanning and notifications
+
+> **Note:** Available from Grafana 10.0.
```

Comment:
We actually don't need to mention this because our docs are versioned. This doc will be released for G10, so no need to mention a specific verison.